Src/AutoFixture/Fixture.cs

```diff
@@ -119,14 +119,17 @@ public Fixture(ISpecimenBuilder engine, MultipleRelay multiple)
                                     new MethodInvoker(
                                         new ModestConstructorQuery()),
                                     new NullableEnumRequestSpecification()),
-                                new RangeAttributeRelay(),
-                                new StringLengthAttributeRelay(),
-                                new RegularExpressionAttributeRelay(),
                                 new EnumGenerator(),
                                 new LambdaExpressionGenerator(),
                                 CreateDefaultValueBuilder(CultureInfo.InvariantCulture),
                                 CreateDefaultValueBuilder(Encoding.UTF8),
-                                CreateDefaultValueBuilder(IPAddress.Loopback))),
+                                CreateDefaultValueBuilder(IPAddress.Loopback),
+                        
+                                /* Data annotations */
```

Comment:
There must be a better way. :)

Reply:
Yep. That is a temporary approach (probably). Later when we have a few more implementations, we could see whether it makes sense to organize them to a kind of standalone builder or so. For now it isn't clear how to deal with that to not loose the flexibility of potential customization... Any way, that change is out of scope of this PR.

Reply:
I've opened https://github.com/AutoFixture/AutoFixture/issues/944 so we don't easily forget it.